src/index.html

```diff
@@ -15,8 +15,65 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="list">
+          <li>
+            <a class="link is-active">Apple</a>
+          </li>
+          <li>
```

Comment:
```suggestion
          </li>
          
          <li>
```
<img width="1030" alt="image" src="https://github.com/user-attachments/assets/a5e76669-f22c-404b-93c2-279852c0cdc5">

The same in all places